packages/openapi-generator/README.md

```diff
@@ -20,38 +20,60 @@ $ npm install @sap-cloud-sdk/openapi-generator
+  -o, --outputDir=<path/to/output>                        (required) Specify the path to the directory to generate the
+                                                          client(s) in. Each client is generated into a subdirectory
+                                                          within the given output directory. Creates the directory if it
+                                                          does not exist. Customize subdirectory naming through
+                                                          `--serviceMapping`.
 
-  --generatePackageJson                        By default, the generator will generate a package.json file, specifying
-                                               dependencies and scripts for compiling and generating documentation. When
-                                               set to false, the generator will skip the generation of the package.json.
+  -t, --transpile                                         Transpile the generated TypeScript code. When enabled a
+                                                          default `tsconfig.json` will be generated. It emits `.js`,
+                                                          `.js.map`, `.d.ts` and `.d.ts.map` files. To configure
+                                                          transpilation set `--tsconfig`.
 
-  --serviceMapping=serviceMapping              Configuration file to ensure consistent names between multiple generation
-                                               runs with updated / changed metadata files. By default it will be read
-                                               from the input directory as "service-mapping.json".
+  --clearOutputDir                                        Remove all files in the output directory before generation. Be
+                                                          cautious when using this option, as it really removes
+                                                          EVERYTHING in the output directory.
 
-  --tsConfig=tsConfig                          tsconfig.json file to overwrite the default "tsconfig.json".
+  --include=<glob/to/include>                             Include files matching the given glob into the root of each
+                                                          generated client directory.
 
-  --versionInPackageJson=versionInPackageJson  By default, when generating package.json file, the generator will set a
-                                               version by using the generator version. It can also be set to a specific
-                                               version.
+  --[no-]packageJson                                      When set to false, no `package.json` is generated. By default,
+                                                          a `package.json` that specifies dependencies and scripts for
+                                                          transpilation and documentation generation.
```

Comment:
verb is missing in `By default, a `package.json` that specifies dependencies and scripts for                                                          transpilation and documentation generation.`